src/driver/driver.cpp

```diff
@@ -188,23 +188,15 @@ void EvolutionDriver::PostExecute(DriverStatus status) {
   Driver::PostExecute(status);
 }
 
-void EvolutionDriver::InitializeBlockTimeStepsAndBoundaries() {
+void EvolutionDriver::InitializeBlockTimeSteps() {
   // calculate the first time step using Block function
   for (auto &pmb : pmesh->block_list) {
     Update::EstimateTimestep(pmb->meshblock_data.Get().get());
   }
   // calculate the first time step using Mesh function
-  pmesh->boundary_comm_map.clear();
-  const int num_partitions = pmesh->DefaultNumPartitions();
-  for (int i = 0; i < num_partitions; i++) {
-    auto &mbase = pmesh->mesh_data.GetOrAdd("base", i);
+  for (auto &partition : pmesh->GetDefaultBlockPartitions()) {
+    auto &mbase = pmesh->mesh_data.Add("base", partition);
     Update::EstimateTimestep(mbase.get());
-    BuildBoundaryBuffers(mbase);
```

Comment:
Sorry, don't completely understand what you are asking here.

I am 95% sure that building the boundary buffers here is unnecessary (and have been for a while), so I removed it in this PR.

Reply:
My linked code to `mesh.cpp` is almost verbatim what you deleted from `driver.cpp`.  My question was just: are we deleting this block code here because this is actually taken care of here in `mesh.cpp`?

Reply:
ah, I see (didn't read carefully enough). Yeah, I think that code in `mesh.cpp` should do what is needed and I think it was strange to have this buffer building tucked away in the driver as well.